src/components/SearchMap.js

```diff
@@ -34,7 +35,11 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites}) => {
                     title={title}
                     description={description}
                     coordinate={coordinate}
-                />
+                >
+
+                    <Icon type='material-community' name='tent' size={25} color='green'/>
+
+                </Marker>
```

Comment:
Marker can take an image or an icon to display a custom element instead of the Apple 'pin'. @amgoodman5 I used the same icon that we have for the `AddSite` tab as a placeholder; feel free to use something else that looks better, or we can leave this for consistency throughout the app. Your call.